.github/workflows/ci.yml

```diff
@@ -76,22 +76,42 @@ jobs:
     needs:
       - changed
     if: ${{ needs.changed.outputs.versions-json-changed == 'true' }}
+    permissions:
+      pull-requests: write
+    outputs:
+      packages-version: ${{ steps.versions.outputs.packages }}
+      config-version: ${{ steps.versions.outputs.config }}
```

Comment:
I guess these are outputs so they can be consumed/used in other workflows? Or do you need to do this even though they're being access in the same workflow in the `Comment` step?

Reply:
Whoops! I'd originally had it as a 1-step job that needed those outputs, but I'd converted it to the last step of that existing workflow. I'll remove them now